src/components/RecentThoughts/RecentThoughts.jsx

```diff
@@ -0,0 +1,72 @@
+import { useEffect } from "react";
+import moment from "moment";
+import "./RecentThoughts.css";
+
+export const RecentThoughts = ({ items, setItems }) => {
+  //Fetching data from the API. Sends a GET request to the API endpoint and expects a JSON response.
```

Comment:
Nice and clean with clear comments! Learning from this and knowing now that my own code would benefit from get inspired from this.